### PR TITLE
Add ledger(product_category) to invoices.info

### PR DESCRIPTION
**Pull request by luisaterrabrito**

### CODE REVIEW

**Comment by lowiebenoot** [comment]
It doesn't make sense to me that we use `product_category_id` directly.

It would make more sense to me if it was like this:

```
product_category (object, nullable)
    type: `product_category` (string)
    id: `e2314517-3cab-4aa9-8471-450e73449041` 
    ledger_account_number: 70100 (number, nullable)
```

Not sure if the `ledger_account_number` should be inside the product category object though.

@jenssegers?

File: src/05-invoicing/invoices.apib [hunk 1]
```diff
@@ -123,6 +123,8 @@ Get details for a single invoice.
                             + product (object, nullable)
                                 + type: `product` (string)
                                 + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
+                            + product_category_id: `e2314517-3cab-4aa9-8471-450e73449041` (string, nullable)
```
Comment (jenssegers) [inline src/05-invoicing/invoices.apib]: This should be an `product_category` object.

 :ok: by @luisabritotl on 13/03/2018 09:10:47

File: src/05-invoicing/invoices.apib [hunk 2]
```diff
@@ -123,6 +123,8 @@ Get details for a single invoice.
...
+                            + ledger_account_number: 70100 (number, nullable)
```
Comment (jenssegers) [inline src/05-invoicing/invoices.apib]: This can go.

 :ok: by @luisabritotl on 13/03/2018 09:10:48

**Review by jenssegers — CHANGES_REQUESTED** [review]

**Review by lowiebenoot — APPROVED** [review]

File: apiary.apib [hunk 3]
```diff
@@ -1444,8 +1444,9 @@ Get details for a single invoice.
                             + product (object, nullable)
                                 + type: `product` (string)
                                 + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
-                            + product_category_id: `e2314517-3cab-4aa9-8471-450e73449041` (string, nullable)
-                            + ledger_account_number: 70100 (number, nullable)
+                            + product_category (object, nullable)
+                                + id: `e2314517-3cab-4aa9-8471-450e73449041` (string)
+                                + type: `product_category` (string)
```
Comment (jenssegers) [inline apiary.apib]: `type` values are camelCased, so should be `productCategory` here.

Comment (jenssegers) [inline apiary.apib]: `type` values are camelCased, so should be `productCategory` here.

**Review by jenssegers — CHANGES_REQUESTED** [review]

**Review by jenssegers — APPROVED** [review]

**Review by jdrieghe — APPROVED** [review]

**Review by lowiebenoot — APPROVED** [review]

**Review by steftrenson — APPROVED** [review]

**Pull request merged**